CMakeLists.txt

```diff
@@ -69,30 +15,25 @@ message( STATUS "CMAKE_CXX_COMPILER_ID: " ${CMAKE_CXX_COMPILER_ID} )
 if ( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang" )
     message( STATUS "C++ Compiler is Clang" )
     set( PPICM_HAS_CLANGXX "TRUE" )
-   # set( CMAKE_CXX_FLAGS "-stdlib=libc++ -std=gnu++11" )
-    set( CMAKE_CXX_FLAGS "-std=c++14" )
-    set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-register -Wno-empty-body -Wno-unused-value" )
-   # add_definitions( -stdlib=libc++ -std=gnu++11 )  # Or -std=c++11
+    set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wno-deprecated-register -Wno-empty-body -Wno-unused-value" )
```

Comment:
Why do we use the current warnings. Are they useful?

Besides consider adding the following warning flags:

-Wall -Wextra -Wno-deprecated -Werror -pedantic -pedantic-errors